docs/usage.md

```diff
@@ -3,64 +3,63 @@
 
 # Usage
 
-Following are the methods to use FortiSOAR Health Assessment
-- Configure FortiSOAR Health Assessment Connector
-- You can use below steps to generate record
-    - **Usage 1**: Health Assessment
-        - Navigate to Health Assessment.
-        - Click on **Assess System Health**.
-        - A Health Assessment Record will be generated. [Click here](#health-assessment) for more details.
-    - **Usage 2**: Configuration Recommendation
-        - Navigate to Health Assessment.
-        - Click on **Recommend System Configuration**.
-        - A Configuration Recommendation Record will be generated. [Click here](#configuration-recommendation) for more details.
+FortiSOAR Health Assessment can be used in following ways
+- **Usage 1**: Health Assessment
+    - Configure FortiSOAR Health Assessment Connector
+    - Navigate to Health Assessment
+    - Click on **Assess System Health**. 
+    - A Health Assessment Record will be generated. [Click here](#health-assessment) for more details
+- **Usage 2**: Configuration Recommendation
+    - Configure FortiSOAR Health Assessment Connector
+    - Navigate to Health Assessment
+    - Click on **Recommend System Configuration**. A Configuration Recommendation Record will be generated. [Click here](#configuration-recommendation) for more details
 - Once record is generated you can apply recommendation generated. [Click Here](#step-to-apply-generated-configuration-recommendations) for step to apply
 
 # Health Assessment
 How to read Health Assessment Record
 - When you navigate to Health Assessment Record's details view. There will be following tabs
```

Comment:
For better illustration purpose, always attach a sample record screenshot and then explain it